frontend/src/app/styles/app.css

```diff
@@ -0,0 +1,5 @@
+.course-input-container {
+  max-width: 50rem;
```

Comment:
use percentage for widths and heights please

Reply:
I did. Tks